web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -14,3 +14,21 @@
  */
 
 declare(strict_types = 1);
+
+/**
+ * Backup the field policy domain node field.
+ */
+function joinup_core_post_update_0107000(&$sandbox) {
+  $queries = [
+    'ALTER TABLE `node__field_policy_domain` RENAME TO `node__field_topic_backup`;',
+    'CREATE TABLE `node__field_policy_domain` LIKE `node__field_topic_backup`;',
+    'ALTER TABLE `node_revision__field_policy_domain` RENAME TO `node_revision__field_topic_backup`;',
+    'CREATE TABLE `node_revision__field_policy_domain` LIKE `node_revision__field_topic_backup`;',
```

Comment:
I wonder why we didn't use here the database API? E.g.:

```php
\Drupal::database()->schema()->renameTable('node__field_policy_domain', 'node__field_topic_backup');
```

It's safer because we might have a different version of MySQL on other environments but the API unifies the usage.